### DISCUSSION TITLE
How To Manipulate this Border Active Selection?

### DISCUSSION THREAD

**Discussion by chaimel** [fabricjs/fabric.js#9625] [General]
I want it to be always at zero degrees when you rotate the object the box or the border should still be at zero degrees

can you teach me how to manipulate it eventhough the object is at any angle the box should still be at zero deg.
![Capture](https://github.com/fabricjs/fabric.js/assets/57937227/6c538fb9-966c-42df-8874-12c7f64d6b46)

**Comment by ShaMan123** [comment]
You will need to override Object#drawBorders and Control#positionHandler
